charts/relay/values.yaml

```diff
@@ -1,5 +1,9 @@
 ## @section Relay
 
+## @param chain.name Name of the chain
+chain:
```

Comment:
lets nest this value under the servicemonitor top level key. Have the relabeling be conditional only if the chain name is set or have a some kind of enabled key for this feature. If the relabeling is conditional if the chain name is set, have the default be an empty string.